graphbolt/CMakeLists.txt

```diff
@@ -73,6 +73,9 @@ endif()
 add_library(${LIB_GRAPHBOLT_NAME} SHARED ${BOLT_SRC} ${BOLT_HEADERS})
 include_directories(BEFORE ${BOLT_DIR}
                            ${BOLT_HEADERS}
+                           # For CXX20 features:
+                           # `std::atomic_ref`, `std::counting_semaphore`
+                           "../third_party/cccl/libcudacxx/include"
```

Comment:
This add dependency to libcudacxx with CPU dgl, not sure about the consequences, let's keep an eye on it.

Reply:
Consider change this CMAKEList in a independent PR and see the result of regression before jumping to the merge of the real code.

Reply:
There will be 0 effect on regression tests sue to the addition of this line in Cmake.

Reply:
The stuff in it are not in the std namespace. It is in a completely new namespace.

Reply:
If you don't use it, then it won't change anything. The namespace is `::cuda::std`, not `std`.